apps/prairielearn/src/middlewares/authn.ts

```diff
@@ -46,7 +45,7 @@ export default asyncHandler(async (req, res, next) => {
 
     const uuid = data.uuid;
 
-    let authnParams = {
+    const authnParams = {
```

Comment:
Nit, there's a `LoadUserAuth` type in `lib/authn` that we could use as a type annotation here.